traffic_ops/install/bin/_postinstall

```diff
@@ -183,6 +183,22 @@ sub generateDbConf {
     return \%todbconf;
 }
 
+sub setCdnConfGoPort {
+    my $cdnConf = shift;
+    if ( exists $cdnConf->{traffic_ops_golang_port} ) {
```

Comment:
This "if statement" needs to be removed because if you run postinstall the second time, then `traffic_ops_golang_port` gets removed because there is an assumption that the cdn.conf will be always regenerated by postinstall